lib/optimizely.rb

```diff
@@ -270,15 +270,27 @@ def is_feature_enabled(feature_flag_key, user_id, attributes = nil)
         # Send event if Decision came from an experiment.
         if decision.source == Optimizely::DecisionService::DECISION_SOURCE_EXPERIMENT
           send_impression(decision.experiment, variation['key'], user_id, attributes)
+          @notification_center.fire_notifications(
+           NotificationCenter::NOTIFICATION_TYPES[:FEATURE_EXPERIMENT],
+           feature_flag_key, user_id, attributes, variation
+          )
         else
           @logger.log(Logger::DEBUG,
                       "The user '#{user_id}' is not being experimented on in feature '#{feature_flag_key}'.")
         end
 
         @logger.log(Logger::INFO, "Feature '#{feature_flag_key}' is enabled for user '#{user_id}'.")
+        audience = nil
```

Comment:
Put all of the logic for firing NotificationCenter::NOTIFICATION_TYPES[:FEATURE_ROLLOUT] in the else block. We only need to fire it when the source is Rollout.

Reply:
Done